packages/vscode-extension/README.md

```diff
@@ -1,64 +1,98 @@
+3. Click **Install**.
+4. Restart Visual Studio Code after installation.
 
-## Installation prerequisites
-The following software is required to work with SuiteCloud extension for Visual Studio Code:
-- Node.js version 14.16.0 LTS
-- Oracle JDK version 11
+See more details about installation in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installing and Setting Up SuiteCloud Extension for Visual Studio Code</a>.
+
+## <font size="6"> <span style="color:gray"> Getting Started </font> </span>
+
+### <font size="5"> <span style="color:gray"> SuiteCloud Extension for Visual Studio Code User Interface </font> </span>
+Refer to the image below to get familiar with the SuiteCloud Extension for Visual Studio Code user interface.
+
+![Overview](resources/VSCodeUiOverview.png)
+
+
+| Element | Description |
+| --- | --- |
+| 1 | Project structure |
+| 2 | Editor area |
+| 3 | Progress view |
+| 4 | SDF logs |
+| 5 | SuiteCloud notifications |
+
+### <font size="5"> <span style="color:gray"> How to Trigger the Command Palette </font>
+Interact through the Command Palette to use the SuiteCloud Extension for Visual Studio Code. The Command Palette allows you to perform several actions in a straightforward way, improving the user experience to the most. To trigger it, use the shortcut, which is Ctrl+Shift+P by default, or click **View** > **Command Palette**. 
 
-Read the detailed list of prerequisites in [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html).
+<p align="center"><a href="#"><img src="resources/CommandPalette.png" alt="Command Palette" width="500" height="600"></a></p>
 
+### <font size="5"> <span style="color:gray"> Setting Up an Account </font> </span>
+To start using the SuiteCloud Extension for Visual Studio Code, you first need to set up the NetSuite accounts you want to customize. The account set up includes adding an account to the extension and setting an account as default. 
 
-## SuiteCloud Extension commands
-The following commands are available through the Command Palette:
-- [Update Single Object with Account Object](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160147382361.html)
-- [Upload File](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160147414469.html)
-- [List Objects](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_160163955513.html)
-- [Setup Account](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147609118.html)
-- [Deploy Project](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147342366.html)
-- [Add Dependency References to the Manifest](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_160147339580.html)
-- [Import Objects]
-- [Import Files]
-- [Create Project]
-- [Create Files]
+To add an account, follow these steps:
+1. Click Ctrl+Shift=P to open the Command Palette.
```

Comment:
I think there is a typo it should be "+" instead of "="
Ctrl+Shift+P

Reply:
Yes, fixing it. Tx!